subgraph/src/ids/proposal.ts

```diff
@@ -0,0 +1,32 @@
+import {bigIntToBytes32} from '../utils/utils';
+import {Address, BigInt, Bytes} from '@graphprotocol/graph-ts';
+
+/**
+ * Generates a unique identifier for a transaction action proposal entity.
```

Comment:
Reading the text, I don't understand what a `TransactionActionsProposalEntity` is about.

Reply:
Since this is coming from the osx subgraph (https://github.com/aragon/osx/blob/c2be413a998c2995f4ed36ca648a5fd9b4284e77/packages/subgraph/schema.graphql#L351C6-L351C32), I moved it to task https://aragonassociation.atlassian.net/browse/OS-839